### PR TITLE
Displayed enum name instead of Job Name and Enum description on pipeline page(#24w5y8a)

### PR DESCRIPTION
**Pull request by disha1202**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/hotwax/job-manager#contribution-guideline)

### CODE REVIEW

File: src/store/modules/job/mutations.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ const mutations: MutationTree <JobState> = {
     [types.JOB_DESCRIPTION_UPDATED] (state, enums) {
         if (enums) {
             enums.forEach((enumId: any) => {
```
Comment (adityasharma7) [inline src/store/modules/job/mutations.ts:32]:
```suggestion
            enums.forEach((enum: any) => {
```

File: src/store/modules/job/mutations.ts [hunk 2]
```diff
@@ -30,7 +30,7 @@ const mutations: MutationTree <JobState> = {
...
-              state.enumIds[enumId.enumId] = enumId.description
+              state.enumIds[enumId.enumId] = enumId
```
Comment (adityasharma7) [inline src/store/modules/job/mutations.ts:33]:
```suggestion
              state.enumIds[enumId.enumId] = enum
```

File: src/store/modules/job/getters.ts [hunk 3]
```diff
@@ -18,7 +18,11 @@ const getters: GetterTree <JobState, RootState> = {
       return state.cached[id]
     },
     getDescription: (state) => (id: string): any => {
```
Comment (adityasharma7) [inline src/store/modules/job/getters.ts:20]:
```suggestion
    getEnumDescription: (state) => (id: string): any => {
```

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/views/Pipeline.vue [hunk 4]
```diff
@@ -25,15 +25,14 @@
...
-                {{ job.jobName }}
+                {{ getEnumName(job.systemJobEnumId) }}
               </ion-label>
               <ion-badge v-if="job.runTime" color="dark" slot="end">{{ timeTillJob(job.runTime)}}</ion-badge>
             </ion-item>
 
-            <!-- Will remove it from comment when description is avaiable -->
-            <!-- <ion-item lines="none">
+            <ion-item lines="none">
               {{ getDescription(job.systemJobEnumId) }}
```
Comment (dt2patel) [inline src/views/Pipeline.vue:35]: Use p tag here inside ion item

**Review by dt2patel — CHANGES_REQUESTED** [review]
Small change

File: src/store/modules/job/mutations.ts (hunk 1 above)
Comment (disha1202) [reply src/store/modules/job/mutations.ts]: Renamed it enumInfo as enum is a keyword.

File: src/store/modules/job/mutations.ts (hunk 2 above)
Comment (disha1202) [reply src/store/modules/job/mutations.ts]: Renamed it enumInfo as enum is a keyword.

File: src/store/modules/job/getters.ts (hunk 3 above)
Comment (disha1202) [reply src/store/modules/job/getters.ts]: Made the requested change.

File: src/views/Pipeline.vue (hunk 4 above)
Comment (disha1202) [reply src/views/Pipeline.vue:35]: Made the requested change

**Review by dt2patel — APPROVED** [review]

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**